spec/latest/core/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
 <title>Hydra Core Vocabulary</title>
 <meta charset="utf-8">
+<script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=fetch|gated"></script>
```

Comment:
Being able to use `Fetch` and `Promise` makes it worthwhile, I think. Or should we require a certain browser version to be able to read the specification?